app/models/purchase.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function(sequelize, DateTypes) {
+  const Purchase = sequelize.define(
+    'Purchase',
+    {
+      id: {
+        type: DateTypes.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+        unique: true
+      },
+      albumId: {
+        type: DateTypes.INTEGER,
+        allowNull: false
+      },
+      date: {
+        type: DateTypes.DATE,
+        defaultValue: DateTypes.NOW
+      }
+    },
+    {
+      timestamps: false
```

Comment:
I would leave the timestamps in true, so you can have more idea of what happened for each row.

Reply:
changed